packages/in_app_purchase/lib/src/store_kit_wrappers/sk_payment_queue_wrapper.dart

```diff
@@ -41,6 +41,12 @@ class SKPaymentQueueWrapper {
     callbackChannel.setMethodCallHandler(_handleObserverCallbacks);
   }
 
+  /// Get pending transactions.
+  Future<List<SKPaymentTransactionWrapper>> get pendingTransactions async {
```

Comment:
Two minor style things here:

1. Dart getters are discouraged for calls that start async work. I think this should probably be a regular method instead. https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#getters-feel-faster-than-methods
2. Should this be named `transactions` instead of `pendingTransactions` to wrap the underlying class better?

Reply:
good points, done